### PR TITLE
Add link to C++ Copilot getting started video

### PR DESCRIPTION
**Pull request by esweet431**
I work on the C++ team at Microsoft. We also have a getting started video for Copilot with C++ (https://www.youtube.com/watch?v=ZfT2CXY5-Dc), I've added the link to mirror other languages

### CODE REVIEW

**Review by Gotunow — APPROVED** [review]

**Review by ntrogh — DISMISSED** [review]
@esweet431 Hey Erica. Great video and thanks for adding it to the Copilot docs. Maybe you could also add it to the [C++ Intro Videos page](https://code.visualstudio.com/docs/cpp/introvideos-cpp)?

**Review by aeschli — APPROVED** [review]

**Comment by esweet431** [comment]
> @esweet431 Hey Erica. Great video and thanks for adding it to the Copilot docs. Maybe you could also add it to the [C++ Intro Videos page](https://code.visualstudio.com/docs/cpp/introvideos-cpp)?

Great idea, added.

File: docs/cpp/introvideos-cpp.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ This video shows you how to customize debug configurations for your C++ project
 
 <iframe src="https://channel9.msdn.com/Shows/Docs-Dev-Tools/Debug-a-C-project-in-VS-Code/player" width="960" height="540" allowFullScreen frameBorder="0" title="Debug a C++ project in VS Code - Microsoft Channel 9 Video"></iframe>
 
+## Getting started using Copilot Chat for C++ in VS Code
+
+This video covers some use cases for using Copilot Chat in VS Code with C++ Code.
+
+<iframe src="https://www.youtube.com/watch?v=ZfT2CXY5-Dc" width="960" height="540" allowFullScreen frameBorder="0" title="Getting Started using Copilot Chat for C++ in VS Code"></iframe>
```
Comment (ntrogh) [inline docs/cpp/introvideos-cpp.md:43]:
```suggestion
<iframe src="https://www.youtube-nocookie.com/embed/ZfT2CXY5-Dc?si=0F2qRniRhN-sg3cR" width="960" height="540" allowFullScreen frameBorder="0" title="Getting Started using Copilot Chat for C++ in VS Code"></iframe>
```

Comment (ntrogh) [reply docs/cpp/introvideos-cpp.md:43]: Using an embed link that doesn't add YT cookies.

**Review by ntrogh — CHANGES_REQUESTED** [review]

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**